pom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <modules>
+        <module>local-storage-service</module>
+    </modules>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
```

Comment:
@carlwilson Can we keep this parent (it comes with typical Spring dependency/plugin mngmt) or better to migrate to verapdf parent?

Reply:
Let's keep it as spring boot for now, later we can update that when we start integrating with Jenkins if needed.